components/ActionSheet/src/Theming/MDCActionSheetController+MaterialTheming.m

```diff
@@ -37,7 +37,13 @@ - (void)applyThemeWithScheme:(id<MDCContainerScheming>)scheme {
 }
 
 - (void)applyThemeWithColorScheme:(id<MDCColorScheming>)colorScheme {
-  self.backgroundColor = colorScheme.surfaceColor;
+  if (colorScheme.elevationOverlayEnabledForDarkMode) {
+    self.backgroundColor = [colorScheme.surfaceColor
+        mdc_resolvedColorWithTraitCollection:self.traitCollection
+                                   elevation:self.view.mdc_absoluteElevation];
+  } else {
+    self.backgroundColor = colorScheme.surfaceColor;
+  }
```

Comment:
This if/else is almost identical to the one below on line 62. These should be refactored into a single function.

Reply:
Refactored to a method.